libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/SetSkillBotFrameworkClientMiddleware.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.Mocks;
+using Microsoft.Bot.Builder.Skills;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Testing
+{
+    /// <summary>
+    /// Middleware to add <see cref="MockSkillBotFrameworkClient"/> to the <see cref="ITurnContext.TurnState"/>.
+    /// </summary>
+    public class SetSkillBotFrameworkClientMiddleware : IMiddleware
```

Comment:
This seems like it will only be useful in skill testing, please move this to the library where the tests are being performed?